flight/goto/mission/goto.py

```diff
@@ -9,20 +9,30 @@
 import logging
 import math
 
-async def move_to(drone: System, latitude: float, longitude: float, altitude: float) -> None:
+async def move_to(drone: System, latitude: float, longitude: float, altitude: float, fast_mode: bool) -> None:
     """
     This function takes in a latitude, longitude and altitude and autonomously
     moves the drone to that waypoint. This function will also auto convert the altitude
-    from feet to meters
+    from feet to meters.
 
-    Args:
-        drone (System): a drone object that has all offboard data needed for computation
-        latitude (float): a float containing the requested latittude to move to
-        longitude (float): a float containing the requested longitude to move to
-        altitude (float): a float contatining the requested altitude to go to (in meters)
+    Parameters
+    ----------
+    drone: System 
+        a drone object that has all offboard data needed for computation
+    latitude: float 
+        a float containing the requested latittude to move to
+    longitude: float 
+        a float containing the requested longitude to move to
+    altitude: float 
+        a float contatining the requested altitude to go to (in feet)
+    slow_mode: bool
+        a boolean that dertermines if the drone will take less time checking its precise location
+        before moving on to another waypoint. If its true, it will move faster, if it is false,
+        it will move at normal speed
 
-    Returns:
-        None
+    Returns
+    -------
+    None
```

Comment:
I read a little bit ago that functions that don't return anything don't need this section in the docstrings.